auto_round/autoround.py

```diff
@@ -147,6 +146,7 @@ def __init__(
             act_sym: bool = None,
             act_dynamic: bool = True,
             quant_block_list: list = None,
+            enable_norm_bias_tuning: bool = False,
```

Comment:
default to None?

Reply:
as the method is not mature, set to False currently